bot/exts/evergreen/realpython.py

```diff
@@ -0,0 +1,75 @@
+# Credit to Vthechamp22, who heavily inspired this bit of code
+import logging
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext import commands
+
+from bot import bot
+from bot.constants import Colours
+
+logger = logging.getLogger(__name__)
+
+API_ROOT = "https://realpython.com/search/api/v1/"
```

Comment:
Are you sure there shouldn't be more empty lines?

Reply:
It passed flake 8, but I will add it real quick